packages/amplify-graphql-relational-transformer/src/__tests__/relational-auth.test.ts

```diff
@@ -70,7 +71,14 @@ test('ModelXConnection type is getting the directives added, when a field has @h
     postUserId: ID! @index(name: "byUser")
     message: String
   }`;
-  const transformer = getTransformer(withAuthModes(iamDefaultConfig, ['AMAZON_COGNITO_USER_POOLS']));
+  const modelToDatasourceMap = new Map<string, DatasourceType>();
+  ['User', 'Post'].map((modelName) => {
+    modelToDatasourceMap.set(modelName, {
+      dbType: 'DDB',
+      provisionDB: true,
+    });
+  });
```

Comment:
Is this now required for DDB datasource? Ideally DDB should be default if mapping is not specified.

Reply:
It shouldn't be required in the end-to-end experience since we construct this map as part of the `loadSchema`. Since this test doesn't go through that flow, I added these. I'll take another look at this test setup.